docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
```

Comment:
The latest version is actually 3, but the structure is mostly the same and 2 is compatible with more tools.